http_check/datadog_checks/http_check/http_check.py

```diff
@@ -318,15 +320,15 @@ def check_cert_expiration(
 
         except Exception as e:
             msg = str(e)
-            if 'expiration' in msg:
+            if any(word in msg for word in ['expired', 'expiration']):
                 self.log.debug("error: %s. Cert might be expired.", e)
                 return AgentCheck.CRITICAL, 0, 0, msg
             elif 'Hostname mismatch' in msg or "doesn't match" in msg:
                 self.log.debug("The hostname on the SSL certificate does not match the given host: %s", e)
-                return AgentCheck.CRITICAL, 0, 0, msg
+                return AgentCheck.CRITICAL, -1, -1, msg
```

Comment:
I think is better to be more explicit with this being an undefined value and use `None`